src/App.css

```diff
@@ -1,38 +1,122 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@1,700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@500&display=swap');
+
+html {
+  background-image: url('./assets/background.jpg');
+  background-repeat: round;
+}
+
 .App {
+  display: flex;
+  flex-direction: column;
+}
+
+.form-and-header {
   text-align: center;
+  border: solid 2px crimson;
+  border-radius: 1rem;
+  box-shadow: 5px 10px crimson;
+  width: 600px;
+  margin: 50px auto;
+  background-color: #F8E6E8;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.form-questions {
+  width: 80%;
+  margin: 0 auto;
+  font-family: 'Quicksand', sans-serif;
+  font-size: 1rem;
+  letter-spacing: 1.5px;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.questions {
+  padding: 0 0 40px 0;
+}
+
+.option-one,
+.option-two {
+  padding: 15px;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.question-options {
   display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
+  justify-content: space-evenly;
+}
+
+legend {
+  padding: 10px 0 10px 0;
+}
+
+h1 {
+  font-family: 'Playfair Display', serif;
+  font-size: 3rem;
+  color: crimson;
+  letter-spacing: 3px;
+}
+
+h2 {
+  font-family: 'Quicksand', sans-serif;
+  letter-spacing: 1px;
+  padding: 0 0 20px 0; 
+}
+
+h3 {
+  font-family: 'Playfair Display', serif;
+  font-size: 2rem;
+  color: crimson;
+  letter-spacing: 3px;
+}
+
+.grow { 
+  transition: all .2s ease-in-out; 
+}
+
+.grow:hover { 
+  transform: scale(1.1); 
+}
+
+.take-the-quiz {
+  letter-spacing: 5px;
+}
+
+button {
```

Comment:
Just a suggestion - would be nice to have some hover and focus effects to let the user know they can interact with the button! Otherwise, love the styling :)

Reply:
I added a pulse effect hehe!